### PR TITLE
SupriyaRudroju.json

### PR DESCRIPTION
**Pull request by supriya855**

### CODE REVIEW

**Review by RakeshSangem — APPROVED** [review]
LGTM🎉

File: data/users/SupriyaRudroju.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Supriya Rudroju",
+  "bio": "Java Dev",
+  "image": "https://avatars.githubusercontent.com/u/83066985?v=4",
```
Comment (codewithvoid) [inline data/users/SupriyaRudroju.json:4]: use some non default image url instead? :)

Comment (supriya855) [reply data/users/SupriyaRudroju.json:4]: okay!

Comment (RakeshSangem) [reply data/users/SupriyaRudroju.json:4]: > okay!

You can use Twitter's pfp link too:)

File: data/users/SupriyaRudroju.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Supriya Rudroju",
   "bio": "Java Dev",
-  "image": "https://avatars.githubusercontent.com/u/83066985?v=4",
+  "image": "https://t.co/GhZcVYXIR5",
```
Comment (codewithvoid) [inline data/users/SupriyaRudroju.json:4]: You have updated GitHub pfp so that is fine. You can revert latest commit!

Sorry for the confusion.

Comment (supriya855) [reply data/users/SupriyaRudroju.json]: Yes, I made changes!

**Review by codewithvoid — APPROVED** [review]
LGTM

**Pull request merged**